protocol/avdl/keybase1/user.avdl

```diff
@@ -217,21 +217,18 @@ protocol user {
     union { null, Time } modifyTime;
   }
 
-  record RecordInfoArg {
-    string reportText;
-    boolean attachMessages;
-  }
-
   record UserBlockArg {
     string username;
     union { null, boolean } setChatBlock;
     union { null, boolean } setFollowBlock;
-    union { null, RecordInfoArg } report;
   }
 
   void setUserBlocks(int sessionID, array<UserBlockArg> blocks);
   array<UserBlock> getUserBlocks(int sessionID, array<string> usernames);
 
+  void reportUser(int sessionID, string username, string reason, string comment,
+    boolean includeTranscript, union { null, string } convID);
```

Comment:
talked briefly with Danny and Max, we are not 100% sure what's the right play is, but this seems like a cleaner design:
- we don't have any screen where you can report multiple people at the same time
- report is tied to conversation ID if want to attach chat transcripts.

that being said, calling `setUserBlocks` where always only at most one of the UserBlockArgs will have report struct seemed weird.

Reply:
The only issue here is that we will not be able to tie the report exactly to particular block action on the backend, if it's a separate API call. But I was not comfortable in having `user/set_blocks` doing too many things either, so separate API calls seem justified.